src/runtimeHelpers.ts

```diff
@@ -62,38 +62,81 @@ export const cacheKeyOptions = {
 
 /**
  * Take in all objects passed to .load(), and bucket them by the non
- * batchKey attributes.
+ * batchKeys(including batchKey and secondaryBatchKey) attributes.
  *
  * We use this to chunk up the requests to the resource.
  *
  * Example:
  * ```js
- * partitionItems([
+ * partitionItems('bar_id', [
  *   { bar_id: 2, include_extra_info: true },
  *   { bar_id: 3, include_extra_info: false },
  *   { bar_id: 4, include_extra_info: true },
- * ], 'bar_id')
+ * ])
  * ```
  *
  * Returns:
  * `[ [ 0, 2 ], [ 1 ] ]`
  *
  * TODO: add generic instead of 'object' for the items array argument
  */
-export function partitionItems(ignoreKey: string, items: ReadonlyArray<object>): ReadonlyArray<ReadonlyArray<number>> {
+export function partitionItems(
+    ignoreKeys: Array<string> | string,
+    items: ReadonlyArray<object>,
+): ReadonlyArray<ReadonlyArray<number>> {
     const groups: {
         [key: string]: Array<number>;
     } = {};
-
```

Comment:
why remove the space?

Adding a space between stanzas is good imo

This is a good example of where different folks have different opinions about style - in such cases, if it's new code that you're writing, go with the style you prefer! But prefer to leave the existing code as is (which is good to minimize the diff you're making anyway)

Unless it's a truly horrible style practice that you want to improve on - in which case great! And maybe we can find an eslint rule or something